__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -13,7 +13,7 @@ describe('<ResourceTemplateForm /> functional testing', () => {
     rtId={'resource:schema:Name'}
     resourceTemplate={ basicRt } />)
 
- const testId = jest.spyOn(shortid, 'generate').mockReturnValue('abcd45')
+ jest.spyOn(shortid, 'generate').mockReturnValue('abcd45')
```

Comment:
If I understand correctly, you are using `jest.spyOn()` here not as a spy but as a stub. Given that, I wonder whether you would consider using `jest.mock()` instead.